src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the backgroundTaskManager class. */
+public interface backgroundTaskManagerInterface { 
+/**
+   * Checks VM status to make sure they are healthy. 
+   */
+    void checkVM();
+/**
+   * Removes attendee if they haven't been polled in 2 minutes.
+   * Nothing associated with if the attendee is the controller.  
+   */
+    void removeAttendee(); 
```

Comment:
deleteInactiveAttendees()

You may need to add support to DatastoreClient for this. Talk to Taniece but you should add the support. Basically youll need to add a function to DatastoreClientInterface to get attendees where lastTimePolled is > some number value.

Reply:
Gotcha